### PR TITLE
first draft of full schedule

### PR DESCRIPTION
**Pull request by aaarendt**
This is a first attempt at the full schedule. Hoping to get something up quickly since participants are requesting it, then we can modify and fill in detail.

### CODE REVIEW

File: book/schedule.yaml [hunk 1]
```diff
@@ -1,38 +1,171 @@
...
+      - time: 10:00 - 10:30
+        location: Maple Hall
+        title: Overview of the SnowEx Mission
+        description:
+        leads:
+          - HP Marshall
+      - time: 10:30 - 10:45
+        title: Movement and coffee break
+        description:
+        leads: [ ]
```
Comment (jomey) [inline book/schedule.yaml]: The template is setup to handle time slots with no leads. With that, there is no need to give an empty list and the key can be removed.

**Comment by jomey** [comment]
The PR changes look good to me. I have some time tomorrow morning to check why the GitHub Actions are not happy right now.

File: book/schedule.yaml [hunk 2]
```diff
@@ -1,38 +1,171 @@
...
-      - list2
-- title: Day 3
-  date: Wed 13, July
-- title: Day 4
-  date: Thurs 14, July
-- title: Day 5
-  date: Friday 17, July
+  - title: Day 1
+    date: Monday 11 Jul
+    contents:
```
Comment (JessicaS11) [inline book/schedule.yaml:5]: looks like the whole file may be over-indented... sorry if this came from my auto-script!

**Comment by JessicaS11** [comment]
> The PR changes look good to me. I have some time tomorrow morning to check why the GitHub Actions are not happy right now.

Missing the jupytext dependency. I'm generating the lock files now and will open a separate PR with the update.

File: book/schedule.yaml [hunk 3]
```diff
@@ -1,38 +1,171 @@
...
+      leads:
+      - Dylan Boyd
+      - Batu Osmanogolu
+    - time: 11:15 - 12:30
+      title: LiDAR
+      leads:
+      - Naheem Adebisi
+      - Clare Webster
+    - time: 12:30 - 1:30
+      title: lunch
```
Comment (jomey) [inline book/schedule.yaml:108]: We have a mix and match of capitalization. 
```suggestion
      title: Lunch
```
Across all time slots

File: book/schedule.yaml (hunk 2 above)
Comment (aaarendt) [reply book/schedule.yaml:5]: This indentation style follows what we had for ICESat-2 and I believe is standard for YAML? Let me know if I missed something.

**Comment by aaarendt** [comment]
@jomey can you give me a lesson on how best to handle the changes added via force-push above? I pull them to my local branch, then recommit?

File: book/schedule.yaml (hunk 2 above)
Comment (jomey) [reply book/schedule.yaml:5]: With our current `.editorconfig` file, we should have two spaces there. So @aaarendt, this is correct what you have.

**Comment by jomey** [comment]
> @jomey can you give me a lesson on how best to handle the changes added via force-push above? I pull them to my local branch, then recommit?

My bad and apologies, I forgot about the fact that my force push will make things more tricky for you. Should have used a merge commit.

If your local copy is 'clean' with no pending changes, do the following:
  1. `git fetch --all`
  1. `git reset --hard origin/schedule`

Let me know if that works for you.

Alternatively, you can also delete your local branch and then checkout anew.
  1. `git checkout master`
  1. `git branch -D schedule`
  1. `git fetch --all`
  1. `git checkout schedule`

**Review by jomey — APPROVED** [review]

File: book/schedule.yaml (hunk 2 above)
Comment (JessicaS11) [reply book/schedule.yaml:5]: My mistake - I was comparing to the template, which is different!

**Pull request merged**